src/Core_Command.php

```diff
@@ -695,9 +710,12 @@ private function _multisite_convert( $assoc_args ) {
 				$cache_key = sprintf( '%d:site_admins', $row->site_id );
 				wp_cache_delete( $cache_key, 'site-options' );
 
-				$result = $wpdb->delete( $wpdb->sitemeta, array(
-					'meta_id' => $row->meta_id,
-				) );
+				$result = $wpdb->delete(
```

Comment:
Single item associative arrays don't need to be multi-line, so this could be rewritten to either of the below:
```php
				$result = $wpdb->delete(
					$wpdb->sitemeta,
					array( 'meta_id' => $row->meta_id )
				);
```

```php
				$result = $wpdb->delete( $wpdb->sitemeta, array( 'meta_id' => $row->meta_id ) );
```